### PR TITLE
implements #143

### LINKED ISSUE #143: Trigger Req: Score >= X
# Feature request

I think adding a trigger where the user's score is X or more could be superb. I had an event where you had to count at least 150 times to be eligible for a giveaway, I was hoping the bot could see if a user has 150 >= and assign a role accordingly.

I understand they may spam triggers because every count from a user that is already above X could trigger actions but I think you could work around it. I specifically want this for role assignment, nothing else but decipher as you see suitable.

### PR DESCRIPTION
**Pull request by wrac4242**
resolves #143 to add a greater than or equal to trigger

### CODE REVIEW

File: src/constants/triggers.ts [hunk 1]
```diff
@@ -52,6 +52,14 @@ const triggers: Record<string, Trigger> = {
     supports: ["flows"],
     limitPerFlow: 1,
   },
+  greater: {
+    short: "Greater than or equal to number X",
+    long: "This will get triggered whenever a user counts a number that is greater than or equal to the number X.",
+    properties: [propertyTypes.numberPositive],
+    explanation: ([[number]]: Array<Array<number>>) => `When someone counts the number ${number} or more`,
+    check: ({ count }, [[number]]: Array<Array<number>>) => Promise.resolve(typeof count === typeof number && count >= number),
```
Comment (promise) [inline src/constants/triggers.ts]: both variables will always be number so this type check is not needed

File: src/constants/triggers.ts [hunk 2]
```diff
@@ -52,6 +52,14 @@ const triggers: Record<string, Trigger> = {
     supports: ["flows"],
     limitPerFlow: 1,
   },
+  greater: {
+    short: "Greater than or equal to number X",
+    long: "This will get triggered whenever a user counts a number that is greater than or equal to the number X.",
+    properties: [propertyTypes.numberPositive],
+    explanation: ([[number]]: Array<Array<number>>) => `When someone counts the number ${number} or more`,
```
Comment (promise) [inline src/constants/triggers.ts]: explanation can rather be something like `When someone counts a number greater than or equal to ${number}, for example ${number}, ${number + 1}, ${number + 2} and so on.`

**Review by promise — CHANGES_REQUESTED** [review]

**Comment by wrac4242** [comment]
changes have been resolved

**Review by promise — APPROVED** [review]
lgtm! ty 😃

**Pull request merged**